opaque.go

```diff
@@ -228,21 +248,50 @@ func (c *Configuration) GetFakeRecord(credentialIdentifier []byte) (*ClientRecor
 
 // DeserializeConfiguration decodes the input and returns a Parameter structure.
 func DeserializeConfiguration(encoded []byte) (*Configuration, error) {
-	if len(encoded) < confLength+2 { // corresponds to the configuration length + 2-byte encoding of empty context
+	if len(encoded) < confIdsLength+2 { // corresponds to the configuration length + 2-byte encoding of empty context
 		return nil, internal.ErrConfigurationInvalidLength
 	}
 
-	ctx, _, err := encoding.DecodeVector(encoded[confLength:])
+	ctx, _, err := encoding.DecodeVector(encoded[confIdsLength:])
```

Comment:
```suggestion
	ctx, offset, err := encoding.DecodeVector(encoded[confIdsLength:])
```

use the returned offset